app/src/main/java/cm/aptoide/pt/timeline/post/PostPresenter.java

```diff
@@ -1,377 +0,0 @@
-package cm.aptoide.pt.timeline.post;
-
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import cm.aptoide.pt.account.AccountAnalytics;
-import cm.aptoide.pt.account.view.AccountNavigator;
-import cm.aptoide.pt.app.view.AppViewFragment;
-import cm.aptoide.pt.crashreports.CrashReport;
-import cm.aptoide.pt.dataprovider.exception.AptoideWsV7Exception;
-import cm.aptoide.pt.dataprovider.model.v7.BaseV7Response;
-import cm.aptoide.pt.logger.Logger;
-import cm.aptoide.pt.navigator.FragmentNavigator;
-import cm.aptoide.pt.navigator.TabNavigator;
-import cm.aptoide.pt.presenter.Presenter;
-import cm.aptoide.pt.presenter.View;
-import cm.aptoide.pt.timeline.post.exceptions.PostException;
-import cm.aptoide.pt.timeline.view.navigation.AppsTimelineTabNavigation;
-import cm.aptoide.pt.view.BackButton;
-import java.util.Collections;
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-import rx.Completable;
-import rx.Observable;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
-
-class PostPresenter implements Presenter, BackButton.ClickHandler {
-
-  private static final String TAG = PostPresenter.class.getSimpleName();
-
-  private static final String UPLOADER_PACKAGE_NAME = "pt.caixamagica.aptoide.uploader";
-
-  private final PostView view;
-  private final CrashReport crashReport;
-  private final PostManager postManager;
-  private final FragmentNavigator fragmentNavigator;
-  private final TabNavigator tabNavigator;
-  private PostAnalytics analytics;
-  private UrlValidator urlValidator;
```

Comment:
Should remove UrlValidator since it is not used

Reply:
deleted